cmd/maestroctl/cmd/plan/planStatus.go

```diff
@@ -0,0 +1,167 @@
+package plan
+
+import (
+	"encoding/json"
+	"fmt"
+	maestrov1alpha1 "github.com/maestrosdk/maestro/pkg/apis/maestro/v1alpha1"
+	"github.com/spf13/cobra"
+	"github.com/xlab/treeprint"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/tools/clientcmd"
+	"log"
+)
+
+func NewPlanStatusCmd() *cobra.Command {
+	statusCmd := &cobra.Command{
+		Use:   "status",
+		Short: "Shows the status of all plans to an particular instance.",
+		Long: `
+	# View plan status
+	maestroctl plan status --instance=<instanceName> --kubeconfig=<$HOME/.kube/config>`,
+		Run: planStatusCmd,
+	}
+
+	statusCmd.Flags().StringVar(&instance, "instance", "", "The instance name available from 'kubectl get instances'")
+	statusCmd.Flags().StringVar(&kubeConfig, "kubeconfig", "", "The file path to kubernetes configuration file; defaults to $HOME/.kube/config")
+	statusCmd.Flags().StringVar(&namespace, "namespace", "default", "The namespace where the instance is running.")
+
+	return statusCmd
+}
+
+func planStatusCmd(cmd *cobra.Command, args []string) {
+
+	instanceFlag, err := cmd.Flags().GetString("instance")
+	if err != nil || instanceFlag == "" {
+		log.Printf("Flag Error: %v", err)
+	}
+
+	mustKubeConfig()
+
+	_, err = cmd.Flags().GetString("kubeconfig")
+	if err != nil || instanceFlag == "" {
+		log.Printf("Flag Error: %v", err)
+	}
+
+	err = planStatus()
+	if err != nil {
+		log.Printf("Error: %v", err)
+	}
+}
+
+func planStatus() error {
+
+	tree := treeprint.New()
+
+	config, err := clientcmd.BuildConfigFromFlags("", kubeConfig)
+	if err != nil {
+		return err
+	}
+
+	//  Create a Dynamic Client to interface with CRDs.
+	dynamicClient, err := dynamic.NewForConfig(config)
+	if err != nil {
+		return err
+	}
+
+	instancesGVR := schema.GroupVersionResource{
+		Group:    "maestro.k8s.io",
+		Version:  "v1alpha1",
+		Resource: "instances",
+	}
+
+	instObj, err := dynamicClient.Resource(instancesGVR).Namespace(namespace).Get(instance, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+	mInstObj, err := instObj.MarshalJSON()
+
+	instance := maestrov1alpha1.Instance{}
+
+	err = json.Unmarshal(mInstObj, &instance)
+	if err != nil {
+		return err
+	}
+
+	frameworkVersionNameOfInstance := instance.Spec.FrameworkVersion.Name
+
+	frameworkGVR := schema.GroupVersionResource{
+		Group:    "maestro.k8s.io",
+		Version:  "v1alpha1",
+		Resource: "frameworkversions",
+	}
+
+	//  List all of the Virtual Services.
+	frameworkObj, err := dynamicClient.Resource(frameworkGVR).Namespace(namespace).Get(frameworkVersionNameOfInstance, metav1.GetOptions{})
+	if err != nil {
+		log.Printf("Error: %v", err)
```

Comment:
Why print this error but not the others?  Let's either print all of them (with some more explanation about when the failure happened) or return it like the rest